### PR TITLE
updates to the GA4GH endpoints for e!82

### PR DESCRIPTION
**Pull request by sarahhunt**

### CODE REVIEW

**Comment by magaliruffier** [comment]
it's been rebased against master with the latest patches, looking good now

File: ensembl_rest_testing.conf [hunk 1]
```diff
@@ -129,6 +129,7 @@ jsonp=1
 
 <Model::ga4gh::variantSet>
    ga_config = __path_to(t/test-genome-DBs/testdata/ga_data/ga_vcf_config.json)__
+   dir       = __path_to(t/test-genome-DBs/testdata/ga_data/)__
```
Comment (avullo) [inline ensembl_rest_testing.conf]: There's a lot of redundancies in this part (i.e. Model::ga4gh) of the configuration file. All the subsections refer to the same ga_config file and their directories point to the same location.
I haven't seen the production configuration file, are all these to be deemed potentially different on production? If not, they could be shared up to the higher level.

File: lib/EnsEMBL/REST/Model/ga4gh/variantSet.pm [hunk 2]
```diff
@@ -78,23 +78,22 @@ sub fetch_sets{
...
+    ## get info descriptions from one of the VCF files    
+    my $meta = $self->get_info($dataSet);
+
+    ## add summary of essential info for meta for the data set from the config
     foreach my $key ("assembly", "source_name", "source_url"){
       my %meta;
       $meta{key}   = $key;
       $meta{value} = $dataSet->$key;
-      push @meta, \%meta;
+      push @{$meta}, \%meta;
```
Comment (avullo) [inline lib/EnsEMBL/REST/Model/ga4gh/variantSet.pm]: Speeding things up a bit:
map { push @{$meta}, { key => $_, value => $dataSet->$_ } qw ( assembly source_name source_url )

**Comment by avullo** [comment]
Nice, looks good to me as well.

**Pull request merged**